lib/stat_tracker.rb

```diff
@@ -34,6 +34,15 @@ def lowest_total_score
   end
 
 #------------LeagueStatistics
+  def count_of_teams
+    teams = []
+    games.each do |game|
+      teams << game.away_team_id unless teams.include? game.away_team_id
+      teams << game.home_team_id unless teams.include? game.home_team_id
```

Comment:
Could use the uniq method here instead to take out duplicates.  To use this, you'd have to have an array of all the teams, so I'm not sure if that would make things more computationally expensive.

Reply:
Updated solution to make use of uniq!